spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -784,6 +969,12 @@ def get_buffered_sdram_per_record(self, variable, vertex_slice):
                 return 0
             size = self.__per_timestep_datatypes[variable].size
             return self._N_BYTES_FOR_TIMESTAMP + size
+        if variable in self.__events_per_core_variables:
+            if variable not in self.__events_per_core_recording:
+                return 0
+            size = self.__events_per_core_datatypes[variable].size
+            return self.__max_rewires_per_ts * (
```

Comment:
If this become max_events_per_ts dict, this would need to change to use the variable indicated.